### ISSUE TITLE
Unix.Unix_error(8, "mkdir", "C:/.../bin/") cs, java, cpp, php build error

### ISSUE THREAD

**Issue by SlavaRa** [HaxeFoundation/haxe#6768]
Windows 10
Haxe 3.4.4
Mingw32, cmd
target cs:
```
haxe -cp src -cs C:/.../bin/ -main Main
Unix.Unix_error(8, "mkdir", "C:/.../bin/")
```
target java:
```
$ haxe -cp src -java C:/.../ -main Main
Unix.Unix_error(8, "mkdir", "C:/.../bin/")
```
target cpp:
```
$ haxe -cp src -cpp C:/.../bin/ -main Main
Unix.Unix_error(8, "mkdir", "C:/.../bin/")
```

**Comment by ibilon** [comment]
Might already be fixed by https://github.com/HaxeFoundation/haxe/commit/e5e08799297045a46106f9aad30237f551d8d118 (https://github.com/HaxeFoundation/haxe/issues/6212)

**Comment by Gama11** [comment]
I don't think the fix made it into patch releases, but it should be in 4.0.0-preview.2.

**Comment by pypmannetjies** [comment]
Having this or a related issue if you are running Haxe from Git Bash on Windows. 

`Unix.Unix_error(21, "open_proc", "haxelib path perf.js")`

The `haxelib path perf.js` resolves to:

```
haxelib path perf.js
C:\<path>\.haxelib\perf,js/1,1,8/src/
-D perf.js=1.1.8
```

The same issue does not occur from a normal command prompt. 

This is on version 4.0.0-preview.2

**Comment by RealyUniqueName** [comment]
> I don't think the fix made it into patch releases, but it should be in 4.0.0-preview.2.

Released in 3.4.6
